Classes/Camera/CameraController.swift

```diff
@@ -1276,7 +1275,18 @@ extension CameraController {
     public func show(media: [(CameraSegment, Data?)]) {
         showPreview = true
         self.segments = media.map({ return $0.0 })
-        self.edits = media.map({ return $0.1 })
+        self.edits = media.map({ (_, data) in
+            if let data = data {
+                do {
+                    return try NSKeyedUnarchiver.unarchivedObject(ofClass: EditorViewController.Edit.self, from: data)
+                } catch let error {
+                    print("Failed to unarchive edit: \(error)")
```

Comment:
Would be great if we can remove the print or replace it with DDLog or similar

Reply:
I replaced this with an assertion failure since it should not happen and we want to catch it in development.